docs/docs-beta/docs/guides/automation/asset-sensors.md

```diff
@@ -1,10 +1,80 @@
 ---
-title: Asset Sensors
-sidebar_position: 50
+title: Scheduling pipelines across projects and jobs with Asset Sensors
```

Comment:
Nit - it's not really 'scheduling,' is it? Since we have a concept called 'schedules' I think this framing could be confusing.

Reply:
Good catch, maybe Executing or Triggering?

Reply:
I think 'trigger' is more in line with the wording we've used elsewhere in automation-focused content